frontend/src/lib/lemon-ui/LemonButton/LemonButton.stories.tsx

```diff
@@ -245,6 +232,47 @@ export const Active = (): JSX.Element => {
     )
 }
 
+export const PseudoStates = (): JSX.Element => {
+    return (
+        <div className="space-y-2">
+            <div className="border rounded p-2">
+                <div>
+                    <h5>TYPE=3D STATE=DEFAULT</h5>
+                    <ButtonVariants3000 />
+                </div>
+                <div id="hover">
+                    <h5>TYPE=3D STATE=HOVER</h5>
+                    <ButtonVariants3000 />
+                </div>
+                <div id="active">
+                    <h5>TYPE=3D STATE=HOVER,ACTIVE</h5>
+                    <ButtonVariants3000 />
+                </div>
+            </div>
+            <div className="border rounded p-2">
+                <div>
+                    <h5>TYPE=TERTIARY STATE=DEFAULT</h5>
+                    <ButtonVariants3000 tertiary />
+                </div>
+                <div id="hover">
+                    <h5>TYPE=TERTIARY STATE=HOVER</h5>
+                    <ButtonVariants3000 tertiary />
+                </div>
+                <div id="active">
+                    <h5>TYPE=TERTIARY STATE=HOVER,ACTIVE</h5>
+                    <ButtonVariants3000 tertiary />
+                </div>
+            </div>
+        </div>
+    )
+}
+PseudoStates.parameters = {
+    pseudo: {
+        hover: ['#hover .LemonButton', '#active .LemonButton'],
+        active: ['#active .LemonButton'],
+    },
```

Comment:
That's pretty cool.

Reply:
Yeah, it works surprisingly well too. It should work even better e.g. you should just be able to do `hover: true` but the antd styles break things so I opted to limit it to specific elements